src/windows-driver-suites/recommended.qls

```diff
@@ -3,77 +3,31 @@
 
 - description: Recommended and required queries for Windows Drivers.
 - import: windows-driver-suites/mustfix.qls
-- queries: .
-  from: microsoft/windows-drivers
-- include:
-    query path: 
-      - drivers/general/queries/AnnotationSyntax/AnnotationSyntax.ql
-      - drivers/general/queries/CurrentFunctionTypeNotCorrect/CurrentFunctionTypeNotCorrect.ql
-      - drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql
-      - drivers/general/queries/DriverEntrySaveBuffer/DriverEntrySaveBuffer.ql
-      - drivers/general/queries/ExaminedValue/ExaminedValue.ql
-      - drivers/general/queries/ExtendedDeprecatedApis/ExtendedDeprecatedApis.ql
-      - drivers/general/queries/IRPStackEntryCopy/IRPStackEntryCopy.ql
-      - drivers/general/queries/ImportantFunctionCallOptimizedOut/ImportantFunctionCallOptimizedOut.ql
-      - drivers/general/queries/ImproperNotOperatorOnZero/ImproperNotOperatorOnZero.ql
-      - drivers/general/queries/InvalidFunctionClassTypedef/InvalidFunctionClassTypedef.ql
-      - drivers/general/queries/InvalidFunctionPointerAnnotation/InvalidFunctionPointerAnnotation.ql
-      - drivers/general/queries/IoInitializeTimerCall/IoInitializeTimerCall.ql
-      - drivers/general/queries/IrqlAnnotationIssue/IrqlAnnotationIssue.ql
-      - drivers/general/queries/IrqlCancelRoutine/IrqlCancelRoutine.ql
-      - drivers/general/queries/IrqlFloatStateMismatch/IrqlFloatStateMismatch.ql
-      - drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql
-      - drivers/general/queries/IrqlNotUsed/IrqlNotUsed.ql
-      - drivers/general/queries/IrqlSetTooHigh/IrqlSetTooHigh.ql
```

Comment:
Are all of these being removed queries that were previously just "experimental" or "available", then got moved to recommended, then got removed as part of a change in timing how/when to promote rules?